tests/unit/tf/examples/test_usecase_retrieval_with_hpo.py

```diff
@@ -1,7 +1,13 @@
+import pytest
 from testbook import testbook
 
 from tests.conftest import REPO_ROOT
 
+try:
+    import optuna  # noqa: F401
+except ImportError:
+    pytest.skip("No module named 'optuna'", allow_module_level=True)
```

Comment:
could this be achieved with `pytest.importorskip` ?

Reply:
Yes, they are equivalent. Changed to `importorskip` in [727f142](https://github.com/NVIDIA-Merlin/models/pull/897/commits/727f142de46a8fcf65827239d94d36c4779505dd).